manifests/install.pp

```diff
@@ -17,17 +17,18 @@
   $package_options        = undef,
 ) {
   include tomcat
+  $_catalina_home = pick($catalina_home, $::tomcat::catalina_home, $name)
```

Comment:
Hi @dploeger.. this would be backwards incompatible, because the pick will always chose the value from `$::tomcat::catalina_home` which is defaulted to the value set in params.pp.